### PR TITLE
Update quantized layer format

### PR DESCRIPTION
**Pull request by Laicheng0830**
### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

File: tensorlayer/layers/convolution/quan_conv_bn.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# /usr/bin/python
+#!/usr/bin/env python3
```
Comment (zsdonghao) [inline tensorlayer/layers/convolution/quan_conv_bn.py]: 这个地方，对么？？

**Comment by Laicheng0830** [comment]
@zsdonghao Done

File: tensorlayer/layers/convolution/quan_conv_bn.py [hunk 2]
```diff
@@ -121,6 +121,10 @@ def __init__(
             )
         )
 
+        if self.in_channels:
```
Comment (zsdonghao) [inline tensorlayer/layers/convolution/quan_conv_bn.py]: 这个以后是不是能全自动？

Comment (zsdonghao) [reply tensorlayer/layers/convolution/quan_conv_bn.py]: 比如，输入到 Layer.__init__ 里面？

Comment (Laicheng0830) [reply tensorlayer/layers/convolution/quan_conv_bn.py]: 应该是可以放layer的初始化里，quantized这部分改完了，暂时可以合并。

**Pull request merged**